client/src/components/PasswordReset.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from "react";
+import NavBar from "./NavBar";
+import { connect } from "react-redux";
+import { resetPassword } from "../actions";
+
+class PasswordReset extends Component {
+  state = {
+    newPassword: "",
+    confirmNewPassword: ""
+  };
+
+  handleFieldChange = event => {
+    this.setState({ [event.target.name]: event.target.value });
+  };
+
+  handleSubmit = event => {
+    event.preventDefault();
+    const urlParams = new URLSearchParams(this.props.location.search);
```

Comment:
I'd love to hear a little more about how this works, I don't think I've used URLSearchParams before.